flagengine/segments/evaluator_test.go

```diff
@@ -297,7 +297,42 @@ func TestIdentityInSegmentPercentageSplit(t *testing.T) {
 			assert.Equal(t, c.expectedResult, result)
 		})
 	}
-	utils.MockSetHashedPercentageForObjectIds(utils.GetHashedPercentageForObjectIds)
+	utils.ResetMocks()
+}
+
+func TestIdentityInSegmentPercentageSplitUsesDjangoID(t *testing.T) {
+	cases := []struct {
+		identity       *identities.IdentityModel
+		expectedResult bool
+	}{
+		{&identities.IdentityModel{
+			DjangoID:          1,
+			Identifier:        "noDjangoID",
```

Comment:
```suggestion
			Identifier:        "withDjangoID",
```

Reply:
Yes, this is true. It bothers me somewhat that we're creating an identity with an identifier of `"noDjangoID"`, for an identity that does have a `DjangoID` value - it looks like a mistake which could be confusing later. 

Can we choose another identifier that is consistent across the 2 identities?

Reply:
Changed to "Test".